src/ripple/protocol/impl/STParsedJSON.cpp

```diff
@@ -204,8 +213,7 @@ static boost::optional<detail::STVar> parseLeaf (
                 }
 
                 ret = detail::make_stvar <STUInt8> (field,
```

Comment:
Same here: we've already checked the range above so a straight `static_cast<unsigned char>` will work.